server/controllers/BookController.js

```diff
@@ -42,6 +42,33 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+* pagination
+* @name pagination
+* @param {Object} request
+* @param {Object} response
+* @returns {JSON} object
+*/
+  static async pagination(request, response) {
```

Comment:
Class name should be capitalized

Reply:
that is not a class is a static method in a class